### PR TITLE
Voter service for report decision

### LINKED ISSUE #7: Rules to exclude Exception/Error from notification
Hi,

I went through several use cases where I simply cannot fix an error so the best move would be to just not send them to Rollbar (as the pricing is on the error volume).

What is the best way to achieve this in your bundle ? I think about a Voter pattern. We could register Voter that decide if the error/exception has to be notified. It is Symfony style, but perhaps this is overcomplicated…

### PR DESCRIPTION
**Pull request by erichard**
I have created 2 voters:

 - HttpExceptionVoter to avoid HttpException notification
 - SameRefererVoter to avoid NotFoundHttpException notification if the HTTP_REFERER and the HTTP_HOST are not equals. Symfony 2.4+ as we need the request_stack.

Fix #7

### CODE REVIEW

**Comment by erichard** [comment]
@megazoll The last commit introduce a `ReportDecisionManager`.

I have created 2 voters:
 - `HttpExceptionVoter` to avoid HttpException notification
 - `SameRefererVoter` to avoid NotFoundHttpException notification if the HTTP_REFERER and the HTTP_HOST are not equals.

By defaulft, both voter are enabled.

File: Voter/ReportVoterInterface.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Staffim\RollbarBundle\Voter;
+
+interface ReportVoterInterface
+{
+    public function vote($exception);
```
Comment (megazoll) [inline Voter/ReportVoterInterface.php]: Probably it reasonable to add typehint ``Exception`` for ``$exception`` argument

**Comment by megazoll** [comment]
@erichard it seems good for me! But it will be perfect to add some tests :)

File: Voter/ReportVoterInterface.php (hunk 1 above)
Comment (erichard) [inline Voter/ReportVoterInterface.php]: I wasn't sure about typeHinting as I'm not very sure about the error to exception conversion.

But if we stick with this strategy, there is some typehint to add…

**Comment by erichard** [comment]
@megazoll Can you merge this PR please ? I have another one to send ;-) I will think about the tests later.

**Comment by megazoll** [comment]
@erichard merged. Thank you!

**Pull request merged**